website/pages/beacon.mdx

```diff
@@ -0,0 +1,26 @@
+# Beacon
+
+## Core Functionality
+
+**Beacon** is a Weave for cohort discovery queries across a federated network of subnodes to the [GA4GH Beacon](https://www.ga4gh.org/product/beacon-api/) standard.
+- Fetch queries from any compatible Upstream Task API 
+- Fderating those queries to any compatible Downstream Task API clients 
+- Aggregate the returned results into a single response, which is sent back Upstream.
+
+## Weave Identity
+
+| Analytical   | Data Movement | Data Egress | Functional                                 | Metadata |
+|--------------|---------------|-------------|--------------------------------------------|----------|
+| Decomposable | Summary       | Manual      | Initiate: Task API and Process: Relay      |          |
+
+## Use in Federated Research
+
+**Beacon** is deployed in a centralised environment to form a connected network of subnodes, each typically running its own Cohort Discovery system. This architecture enables distributed datasets to be queried as though they were one, without sharing patient-level data.
+
+By aggregating results centrally, institutions can collaborate on large-scale research while maintaining data sovereignty and privacy.
+
+## Reference Implementation
+
+The open-source, reference implementation is **Relay**, which provides a containerised and security-conscious—enabling ethical, scalable health research through federated collaboration. 
```

Comment:
Is Relay the whole reference implementation?

I would think it's:
    - Relay (central)
    - Bunny (federated nodes)

similar to how 5ST (simplified) is:
    - Submission (central)
    - Agent, Egress, TES (federated nodes)